Source/Basic-Conditions-And-Behaviors/Runtime/Conditions/PositionalConditions/ObjectInColliderCondition.cs

```diff
@@ -47,16 +58,7 @@ public class EntityData : IObjectInTargetData
             /// <inheritdoc />
             [HideInProcessInspector]
             [IgnoreDataMember]
-            public string Name
-            {
-                get
-                {
-                    string targetObject = TargetObject.IsEmpty() ? "[NULL]" : TargetObject.Value.GameObject.name;
-                    string triggerProperty = TriggerProperty.IsEmpty() ? "[NULL]" : TriggerProperty.Value.SceneObject.GameObject.name;
-
-                    return $"Move {targetObject} in collider {triggerProperty}";
-                }
-            }
+            public string Name => $"Move {TargetObjects} in collider {TriggerObject}";
 
             /// <inheritdoc />
 #if CREATOR_PRO
```

Comment:
We don't. I left it there in case some day we were allowed to use the Pro code, but it doesn't look like it and it's probably obsolete anyway. I'll remove those.